raft/src/test/java/org/apache/kafka/raft/LeaderStateTest.java

```diff
@@ -447,6 +452,38 @@ public void testDescribeQuorumWithObservers() {
             observerState);
     }
 
+    @Test
+    public void testMajorityFollowerFetchTimeoutExpiration() {
+        int node1 = 1;
+        int node2 = 2;
+        int node3 = 3;
+        int node4 = 4;
+        int observer5 = 5;
+        LeaderState<?> state = newLeaderState(mkSet(localId, node1, node2, node3, node4), 0L);
+        assertFalse(state.hasMajorityFollowerFetchExpired(time.milliseconds()));
+        int resignLeadershipTimeout = (int) (fetchTimeoutMs * 1.5);
```

Comment:
This applies to the other tests but I am thinking that we should either 1) make `1.5` a constant and use that constant in all of the tests or 2) add a method that returns the "check quorum" timeout ms and use that in all of the tests.

Reply:
Agree! If we need to update the `1.5` factor, we can change in one place. Updated.